ui/src/actions/documentActions.js

```diff
@@ -3,24 +3,23 @@ import asyncActionCreator from './asyncActionCreator';
 
 export const ingestDocument = asyncActionCreator(
   (collectionId, metadata, file, onUploadProgress, cancelToken) => async () => {
-    const formData = new FormData();
-    if (file) {
-      formData.append('file', file);
-    }
-    formData.append('meta', JSON.stringify(metadata));
+    const formData = {
+      meta: JSON.stringify(metadata),
+      file: file,
+    };
+
```

Comment:
Can you explain why we're removing the if statement here?

Reply:
The relevant difference is that if `file` has a falsy value, the request would previously not include the `file` field at all, while it would now include the falsy value for the `file` field.

Both cases (omitting the `file` field and transmitting an falsy value) are considered invalid and the backend API will raise the same error in both cases. As it doesn’t make any difference, we can simply remove the if statement and slightly simplify the code.